core/src/main/resources/error/error-classes.json

```diff
@@ -961,6 +961,11 @@
     ],
     "sqlState" : "42000"
   },
+  "NOT_A_PARTITIONED_TABLE" : {
+    "message" : [
+      "Operation <operation> is not allowed for <tableIdentWithDB> because it is not a partitioned table."
```

Comment:
`qualifiedTableName` is better than `tableIdentWithDB` but it's minor.

Reply:
Let me handle this when I submit another error message improvement PR.
Thanks!